server/types/oid/oid.go

```diff
@@ -30,6 +30,7 @@ const (
 	Section_Operator                  // Refers to operators (+, -, *, etc.)
 	Section_Sequence                  // Refers to sequences
 	Section_Table                     // Refers to tables
+	Section_Type                      // Refers to types
```

Comment:
We probably don't need this just yet. We'll need it when we add custom types, but for now all of our types are just the built-in ones, so we can remove this altogether. That means that the `Section` check just needs to be on `BuiltIn`.